core/src/main/scala/org/apache/spark/rdd/PipedRDD.scala

```diff
@@ -205,6 +206,9 @@ private[spark] class PipedRDD[T: ClassTag](
       private def propagateChildException(): Unit = {
         val t = childThreadException.get()
         if (t != null) {
+          val commandRan = command.mkString(" ")
```

Comment:
Nit "command run" instead of "command ran". Here and above you can consistently make a `val` and interpolate it. LGTM